python_modules/dagster/dagster_tests/core_tests/test_op_invocation.py

```diff
@@ -223,17 +224,17 @@ def op_requires_config(context):
     with pytest.raises(
         DagsterInvalidInvocationError,
         match=(
-            "Compute function of op 'op_requires_config' has context argument, but no "
+            "Decorated function 'op_requires_config' has context argument, but no "
             "context was provided when invoking."
         ),
     ):
         op_requires_config()
 
-    # Ensure that alias is accounted for in error message
+    # alias still refers back to decorated function
```

Comment:
could be convinced otherwise on this call

Reply:
I don't mind it

Reply:
Why not both? "Decorated function 'op_requires_config' (aliased to 'aliased_op_requires_config')..."